### DISCUSSION TITLE
Invalid Config

### DISCUSSION THREAD

**Discussion by Fractal-Tess** [leftwm/leftwm#1184] [General]
After doing paru update on my arch machine yesterday, leftwm now refuses to laod config.
Checking what is wrong with `leftwm check` shows that the layouts config option is now taking a string.
![image](https://github.com/leftwm/leftwm/assets/75957529/80a7a91b-cd60-46c8-9ef9-499474ce1331)

Is there any reason that config has to be a string from now on? Also wouldn't this brick leftwm on most users who are on rolling update distros?

**Comment by Eskaan** [comment]
We did a breaking update from 0.4.x to 0.5.1 a few days ago, where we integrated the new layout_definitions library.
A broken config doesn't brick leftwm, it just falls back to the default config (with default keybinds).
